wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/ConversationListTopBarView.swift

```diff
@@ -0,0 +1,155 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UIKit
+
+final class TopBar: UIView {
```

Comment:
I extracted this class from [ConversationListTopBarView.swift](https://github.com/wireapp/wire-ios/pull/979/files#diff-85758b93c193f5ea5105f37e67b9f357f2671567e71efce25b0fe1666cb745b3).